packages/conventional-changelog-eslint/conventional-recommended-bump.js

```diff
@@ -11,14 +11,12 @@ module.exports = {
     let features = 0
 
     commits.forEach(commit => {
-      if (!commit.type) {
-        return
-      }
+      if (!commit.tag) return
```

Comment:
Under what conditions a commit has a "tag" property?

Reply:
[`conventional-changelog-eslint` ](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint) uses "tag" instead of "type" as first array entry in `headerCorrespondence` (see also the preset’s [parser-opts.js](https://github.com/conventional-changelog/conventional-changelog/blob/conventional-changelog-eslint%403.0.9/packages/conventional-changelog-eslint/parser-opts.js) and this PR’s description).

I think that’s not a mistake. The fix here only affects the ESLint preset, so you only have a "tag" property when using the ESLint preset. And not, for example, with the default preset (conventional).

Reply:
@ingmarh Got you. I was under the impression "parsed" commits properties are the same, regardless of the preset.